src/Utils/StringUtils.ts

```diff
@@ -18,12 +18,22 @@ export class StringUtils {
     public static normalize(value: any, defaultValue?: any): any {
         if (value) {
             if (typeof value === 'number') {
-                return value < 0 ? 0 : value;
+                return value;
             }
             if (typeof value === 'string') {
                 return value.toLocaleLowerCase();
             }
         }
         return defaultValue;
     }
+
+    public static parseNumber(value: any): any {
+        if (value) {
+            let result = parseInt(value, 10);
+            if (typeof result === 'number') {
```

Comment:
This method is meaningless, 
- pass a number, it will return the same number, 
- pass a string, it will return `NaN`
- pass a true, it will return `NaN`
- pass a false, it will return `false`

Reply:
pass a "2", it will return 2,